lib/boundary/issue-trackers/linear.ts

```diff
@@ -193,9 +194,13 @@ export class Linear implements IssueTracker {
 
 		const description = await this._renderDescription(incident, log);
 
+		const priorityPrefix = this.#getPriorityPrefix(priority);
+		const projectName = `${priorityPrefix}${this.#config.breakingRoomPrefix}${incident.id}`;
```

Comment:
Should we do this as well?

```suggestion
		const projectName = `${priorityPrefix}${this.#config.breakingRoomPrefix}${incident.id}: ${incident.title}`;
```

And then remove `description` from projectPayload below.